test/Serilog.Settings.Configuration.Tests/ConfigurationSettingsTests.cs

```diff
@@ -194,6 +194,39 @@ public void TestMinimumLevelOverrides()
             Assert.NotNull(evt);
         }
 
+        [Fact]
+        public void TestMinimumLevelOverridesForChildContext()
+        {
+            var json = @"{
+                ""Serilog"": {
+                    ""MinimumLevel"" : {
+                        ""Default"" : ""Warning"",
+                        ""Override"" : {
+                            ""System"" : ""Warning"",
+                            ""System.Threading"": ""Debug""
+                        }
+                    }        
+                }
+            }";
+
+            LogEvent evt = null;
+
+            var log = ConfigFromJson(json)
+                .WriteTo.Sink(new DelegatingSink(e => evt = e))
+                .CreateLogger();
+
+            log.Write(Some.DebugEvent());
+            Assert.Null(evt);
+
+            var custom = log.ForContext("SourceContext", "System.Threading.Tasks.Task<42>");
+            custom.Write(Some.DebugEvent());
+            Assert.NotNull(evt);
+
+            var systemThreadingLogger = log.ForContext<System.Threading.Tasks.Task>();
```

Comment:
maybe you would need to reset `evt` back to `null` to make the last assertion really valid ?

Reply:
will check.